coda/templates/mdstore/about.html

```diff
@@ -35,7 +35,8 @@ <h3>Search API</h3>
     ]
 
     # to get the events which failed after a certain date, append a 'start_date' argument
-    $ curl {{ request.META.HTTP_HOST }}/event/search.json?outcome=http://purl.org/net/untl/vocabularies/eventOutcomes#failure&start_date=05/13/2013
+    $ curl {{ request.META.HTTP_HOST }}/event/search.json?event_outcome=http://purl.org/net/untl/vocabularies/eventOutcomes/%23failure&start_date=05/13/2013
```

Comment:
Trying this one with `curl`, I needed to put the url into quotes because of the `&` before `start_date` which otherwise is interpreted to run the command in the background.